napari/_qt/qt_main_window.py

```diff
@@ -1331,7 +1329,7 @@ def _screenshot(
             Flag to indicate whether flash animation should be shown after
             the screenshot was captured.
         size : tuple (int, int)
-            Size (resolution) of the screenshot. By default, the currently displayed size.
+            Size (resolution Height x Width) of the screenshot. By default, the currently displayed size.
```

Comment:
```suggestion
            Size (resolution height x width) of the screenshot. By default, the currently displayed size.
```

As in te rest of code